crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -111,6 +111,12 @@ impl C8yMapperConfig {
             "tedge/events/+/+",
             "tedge/health/+",
             "tedge/health/+/+",
+            "te/device/+/+/+/m/",
+            "te/device/+/+/+/m/+",
+            "te/device/+/+/+/a/",
+            "te/device/+/+/+/a/+",
+            "te/device/+/+/+/e/",
+            "te/device/+/+/+/e/+",
```

Comment:
There is no points subscribing to `te/device/+/+/+/m/` and `te/device/+/+/+/m/+` as the latter includes the former. Furthermore, one should not assume the first identifier part to be the `device` constant.

```suggestion
            "te/+/+/+/+/m/+",
            "te/+/+/+/+/a/+",
            "te/+/+/+/+/e/+",
```

There is an ambiguity though. When a measurement has no type, do we have to publish it on `te/device/main///m` or `te/device/main///m/` ? I would prefer the latter to be more systematic and to avoid confusion with no vs empty measurement types.

Reply:
Yes if no type is provided, it should be published on `te/device/main///m/`.